tools/deployment/k8s/s3/sample-resource-files/aws-credential.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aws-credential
+  namespace: default
+type: Opaque
+data:
+  credential: W2RlZmF1bHRdCmF3c19hY2Nlc3Nfa2V5X2lkID0gbWluaW9hZG1pbgphd3Nfc2VjcmV0X2FjY2Vzc19rZXkgPSBtaW5pb2FkbWluCg==
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)